vcr/vcr.go

```diff
@@ -137,7 +137,10 @@ func (c *vcr) Configure(config core.ServerConfig) error {
 
 	// Create the JSON-LD Context loader
 	allowExternalCalls := !config.Strictmode
```

Comment:
doesn't deserve this its own config? Why would we enable this on default (in dev/test), since we know beforehand what VCs (and thus contexts) there are going to be?

Reply:
Do we now that? When you sent me a credential with an unknown context I have to look it up. During test/dev this can be automatic, in prod this should be denied.

Reply:
Will your node break if I send you a crafted VC with a schema URL that will send back 1 byte a second (death by slow response)?

(juicy XML Schema location attacks)